org.eclipse.lsp4j/src/main/xtend-gen/org/eclipse/lsp4j/WorkspaceFolder.java

```diff
@@ -26,6 +30,18 @@
    */
   private String name;
```

Comment:
In the spec it is not marked as optional. Is it bug there?

Reply:
The LSP documentation says _"The name of the workspace folder. Defaults to the uri's basename."_ I think it's a bug in the specification: either the property should be optional, or the second part of the property documentation should be removed.